designer/router.js

```diff
@@ -1,8 +1,10 @@
 var wda = require('./controller');
 
 module.exports = function(router) {
-    router.get('/blueprint/:project_id',wda.getBlueprint);
     router.get('/blueprints',wda.getBlueprints);
     router.delete('/delete/:project_id', wda.deleteBlueprint);
     router.get('/projects', wda.getProjectNames);
+    router.post('/update/:project_id',wda.updateBlueprint);
```

Comment:
so that i missed to point out 
but please change this url to 
/blueprints/update